Add user friendly error message

pedrochevallier opened the pull request.

pivotal-cla commented:
@pedrochevallier Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-petclinic&pullRequestId=1483)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-petclinic&pullRequestId=1483) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

pivotal-cla commented:
@pedrochevallier Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-petclinic&pullRequestId=1483)!

pedrochevallier commented:
I just realized it's the same as issue #1442, I removed the leading zeros in the tests and added the message for when the number doesn't have 10 digits.
